charts/workflow/values.yaml

```diff
@@ -113,22 +113,20 @@ controller:
   # admin_only - allows for registration by an admin only.
   registration_mode: "enabled"
 
-database:
-  # Configure the following ONLY if using an off-cluster PostgreSQL database
-  postgres:
-    name: "database name"
-    username: "database username"
-    password: "database password"
-    host: "database host"
-    port: "database port"
-
-logger:
-  redis:
-    # Configure the following ONLY if using an off-cluster Redis instance for logger
-    db: "0"
-    host: "redis host"
-    port: "redis port"
-    password: "redis password" # "" == no password
+# Configure the following ONLY if using an off-cluster PostgreSQL database
+postgres:
+  name: "database name"
```

Comment:
Oh, so it's already namespaced by the chart name? That makes some sense... But why wouldn't off-cluster redis parameters get populated before deis/redis#10 then?